src/bidi.js

```diff
@@ -167,17 +169,27 @@ function applyArabicRequireLigatures() {
  * Apply required arabic ligatures
  */
 function applyLatinLigatures() {
-    const script = 'latn';
-    if (!this.featuresTags.hasOwnProperty(script)) return;
-    const tags = this.featuresTags[script];
-    if (tags.indexOf('liga') === -1) return;
+    if (!this.hasFeatureEnabled('latn', 'liga')) return;
```

Comment:
Is there a reason for this change?

Reply:
The hasFeatureEnabled() function was added with this PR and I guess the motivation was that it's easier to understand for contributors what's happening there. It's fine by me!

Reply:
This bit only got refactored, motivated:
- (primary) It was reasonable to not introduce another redundant code, and keep code DRY, with this trivial change
- (secondary) more comprehend code source